package.json

```diff
@@ -28,9 +30,10 @@
     "url": "https://github.com/resin-io/resin-preload/issues"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "pretest": "standard lib/preload.js bin/resin-preload",
+    "test": "echo done"
```

Comment:
You can omit the `test` script here, and it will just echo 'Error: no test specified' but still exit with a zero status code